src/main/java/fr/openent/homeworkAssistance/models/FeaturedResources.java

```diff
@@ -0,0 +1,70 @@
+package fr.openent.homeworkAssistance.models;
+
+public class FeaturedResources {
+    private String idRessource;
+    private String nomRessource;
+    private String urlVignette;
+    private String urlAccesRessource;
+    private String description;
+
+    public FeaturedResources(String idRessource, String nomRessource, String urlVignette, String urlAccesRessource, String description) {
+        this.idRessource = idRessource;
+        this.nomRessource = nomRessource;
+        this.urlVignette = urlVignette;
+        this.urlAccesRessource = urlAccesRessource;
+        this.description = description;
+    }
+
+    // Getters and Setters
+    public String getIdRessource() {
+        return idRessource;
+    }
+
+    public void setIdRessource(String idRessource) {
+        this.idRessource = idRessource;
+    }
```

Comment:
par convention sur le projet ENT on return l'objet lui-même pour tous les setters :
```suggestion
    public FeaturedResources setIdRessource(String idRessource) {
        this.idRessource = idRessource;
        return this;
    }
```